offersrpc/offersrpc.proto

```diff
@@ -22,3 +24,40 @@ message SendOnionMessageRequest {
 
 message SendOnionMessageResponse {
 }
+
+message DecodeOfferRequest {
+    // The encoded offer string to be decoded.
+    string offer = 1;
+}
+
+message DecodeOfferResponse {
```

Comment:
Have an offer message type here rather, but don't make the function directly return the type, so that there's flexibility.